src/log/log.c

```diff
@@ -1504,6 +1522,27 @@ __wt_log_acquire(WT_SESSION_IMPL *session, uint64_t recsize, WT_LOGSLOT *slot)
 	return (0);
 }
 
+/*
+ * __log_truncate_file_requires_hot_backup_lock --
+ *	Helper for __log_truncate_file. Intended to be called while holding
+ *	the hot backup read lock.
```

Comment:
And here too for the `_int` naming suggestion.

Reply:
bc1914f